Sources/TuistKit/ProjectEditor/ProjectEditor.swift

```diff
@@ -30,12 +30,14 @@ protocol ProjectEditing: AnyObject {
     /// Generates an Xcode project to edit the Project defined in the given directory.
     /// - Parameters:
     ///   - editingPath: Directory whose project will be edited.
-    ///   - onlyCurrentDirectory: True if only the manifest in the current directory should be included.
+    ///   - excluding: Relative glob patterns for excluded files.
     ///   - destinationDirectory: Directory in which the Xcode project will be generated.
+    ///   - onlyCurrentDirectory: True if only the manifest in the current directory should be included.
     ///   - plugins: The plugins to load as part of the edit project.
     /// - Returns: The path to the generated Xcode project.
     func edit(
         at editingPath: AbsolutePath,
+        excluding: [String],
```

Comment:
It there a reason why this has to be part of the editor's interface? If not, I suggest removing it from here and codifying the behavior into the editing logic. If we need that flexibility in the future, we can provide an API for that.

Reply:
👍